sao_paulo/src/main/java/org/eqasim/sao_paulo/mode_choice/SaoPauloModeAvailability.java

```diff
@@ -18,7 +18,7 @@
 		// Modes that are always available
 		modes.add(TransportMode.walk);
 		modes.add(TransportMode.pt);
-		modes.add(TransportMode.bike);
+		//modes.add(TransportMode.bike);
```

Comment:
Is this intended? @balacmi

Reply:
Yes this is intended as we do not have enough observations in HTS at the moment